src/panels/lovelace/components/hui-conditional-base.ts

```diff
@@ -119,7 +96,7 @@ export class HuiConditionalBase extends ReactiveElement {
       changed.has("hass") ||
       changed.has("preview")
     ) {
-      this._listenMediaQueries();
+      this.setupConditionalListeners();
```

Comment:
The listeners are not cleared here. There should be a `clearConditionalListeners()` here or inside `setupConditionalListeners()`